release/models/platform/openconfig-platform-pipeline-counters.yang

```diff
@@ -1197,6 +1204,130 @@ module openconfig-platform-pipeline-counters {
     }
   }
 
+  grouping control-plane-traffic-counters-state {
+    description
+      "Control plane traffic counter state grouping.";
+
+    leaf queued-aggregate {
+      type oc-yang:counter64;
+      description
+        "This captures the aggregation of all counters where the switch has enqueued
+        traffic related to the control-plane.";
+    }
+
+    leaf queued-bytes-aggregate {
+      type oc-yang:counter64;
+      description
+        "This captures the aggregation of all counters in bytes where the switch has
+        enqueued traffic related to the control-plane.";
+    }
+
+    leaf dropped-aggregate {
+      type oc-yang:counter64;
+      description
+        "This captures the aggregation of all counters where the switch has dropped
+        traffic related to the control-plane.";
+    }
+
+    leaf dropped-bytes-aggregate {
+      type oc-yang:counter64;
+      description
+        "This captures the aggregation of all counters in bytes where the switch has
+        dropped traffic related to the control-plane.";
+    }
+  }
+
+  grouping control-plane-traffic-vendor-counters {
+    description
+      "A utility grouping for vendors to use when augmenting the vendor-specific
+      control-plane traffic container.";
+
+    leaf queued {
+      type oc-yang:counter64;
+      description
+        "This counter counts the number of packets enqueued.
+
+        This counter should contribute to the total aggregate of
+        .../pipeline-counters/control-plane-traffic/state/total-queued.";
```

Comment:
Probably need to change total-queued to queued-aggregate, following the rename in [2614b5a429f352175e4ffbf6bd0c0ec119d18771](https://github.com/openconfig/public/pull/763/commits/2614b5a429f352175e4ffbf6bd0c0ec119d18771).

Same on the similar attributes below.